### PR TITLE
Refactor(cb2-4805): added log levels and more logs to EVL feed

### PR DESCRIPTION
**Pull request by ghost**
Pr adds log levels and more logs in general that *hopefully* are more useful for debugging purposes.

### CODE REVIEW

File: src/infrastructure/api/index.ts [hunk 1]
```diff
@@ -113,20 +114,25 @@ router.get(
...
       .then((result) => {
-        console.log('Generating EVL File Data');
+        logger.info('Generating EVL File Data');
         const evlFeedProcessedData: string = result.map(
           (entry) => `${entry.vrm_trm},${entry.certificateNumber},${moment(entry.testExpiryDate).format('DD-MMM-YYYY')}`,
         ).join('\n');
+        logger.debug(`\nData captured for file generation: ${evlFeedProcessedData} \n\n`);
 
         uploadToS3(evlFeedProcessedData, fileName, () => {
+          logger.debug(`Successfully uploaded ${fileName} to S3`);
```
Comment (Ian-Hodges) [inline src/infrastructure/api/index.ts:135]: I think this would be a good as info rather than debug?

File: src/utils/logger.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+const { printf } = format;
+
+const logFormat = printf((info) => {
+  // Checks if log is an error - has stack info
+  if (info.stack) {
+    return `${info.level}: ${info.stack as string}`;
+  }
+  return `${info.level}: ${info.message as string}`;
+});
```
Comment (tahayazici) [inline src/utils/logger.ts:11]:
```Suggestion
// Checks if log is an error - has stack info
const logFormat = printf(info => info.stack
  ? `${info.level}: ${info.stack as string}`
  : `${info.level}: ${info.message as string}`
);
```

**Review by aaron-downing — DISMISSED** [review]

File: src/infrastructure/api/index.ts (hunk 1 above)
Comment (tom-bjss) [reply src/infrastructure/api/index.ts:135]: Can you add it as a suggestion please 😄

Comment (tom-bjss) [reply src/infrastructure/api/index.ts:135]: Done

**Review by Ian-Hodges — APPROVED** [review]

**Review by aaron-downing — APPROVED** [review]

**Pull request merged**